src/biogeochem/CNBalanceCheckMod.F90

```diff
@@ -217,6 +255,54 @@ subroutine CBalanceCheck(this, bounds, num_soilc, filter_soilc, &
          call endrun(msg=errMsg(sourcefile, __LINE__))
       end if
 
+      ! Repeat error check at the gridcell level
+      ! The column-level error gets added to the gridcel-level error below
+      call c2g( bounds = bounds, &
+         carr = col_errcb(bounds%begc:bounds%endc), &
+         garr = grc_errcb(bounds%begg:bounds%endg), &
+         c2l_scale_type = 'unity', &
+         l2g_scale_type = 'unity')
+
+      err_found = .false.
+      do g = bounds%begg, bounds%endg
+         ! calculate gridcell-level carbon storage for mass conservation check
+         grc_endcb(g) = 0._r8  ! TODO Add gridcell terms not accted at col level
+
+         ! calculate total gridcell-level inputs
+         grc_cinputs = 0._r8  ! TODO Add gridcell terms not accted at col level
+
+         ! calculate total gridcell-level outputs
+         grc_coutputs = 0._r8  ! TODO Add gridcell terms not accted at col level
+
+         ! calculate the total gridcell-level carbon balance error
+         ! for this time step
+         grc_errcb(g) = (grc_cinputs - grc_coutputs) * dt - &
+                        (grc_endcb(g) - grc_begcb(g)) + grc_errcb(g)
```

Comment:
In reference to my previous [comment](https://github.com/ESCOMP/CTSM/pull/984/commits/6ba026c04ad1a15b5a5a3675c9059995e218a7c8#r411050340),
this where I picture using all the gridcell-level terms not accounted for at the column level PLUS the column-level error converted to the gridcell level. Pls let me know if you think that this is the wrong way to think about it @billsacks 

Either way, I have encountered a small problem at this stage: 
the gridcell-level error calculated just from
`call c2g( bounds = bounds, carr = col_errcb(bounds%begc:bounds%endc), garr = grc_errcb(bounds%begg:bounds%endg), c2l_scale_type = 'unity', l2g_scale_type = 'unity')`
(all other terms set to 0) exceeds the 1e-7 threshold (largest value 8.6E-5) and aborts. Does this just reflect the level of accuracy of the c2g calculation? I assume so.